charts/gardener/controlplane/charts/application/templates/clusterrole-seeds.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if not .Values.global.rbac.seedAuthorizer.enabled }}
```

Comment:
If the helm chart is applied via `helm template ... | kubectl apply -f -`, the `ClusterRole{Binding}` needs to be removed manually, is that right?

And BTW: why is the `seedAuthorizer` under `rbac`? It has nothing to do with RBAC...